usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -209,4 +219,23 @@ def matview_search_filter(filters, model):
                 or_queryset |= Q(extent_competed__exact=v)
             queryset = queryset.filter(or_queryset)
 
+        # Federal Account Filter
+        elif key == "federal_account_ids":
+            faba_flag = True
+            faba_queryset &= faba_queryset.filter(treasury_account__federal_account_id__in=value)
```

Comment:
It is possible to have these as Q objects and then combine them in the `if faba_flag:` statement? If so then It would be nicer to move `faba_queryset = FinancialAccountsByAwards.objects.filter(award__isnull=False)` into the if statement as well.

Reply:
edited!